_posts/2020-03-28-Server-Rpi.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+title: "Server Installation – Raspberry Pi"
```

Comment:
Why do we have this page when we have https://jamulus.io/wiki/Server-Rpi?

Reply:
Since at least in my opinion the installation on Raspberry Pi is something quite specific we should move to the KB since we already cover a lot of it with the Linux install page.

Reply:
Ah OK, so we would delete https://jamulus.io/wiki/Server-Rpi? In which yes, I would agree.